python/default_crab_config.py

```diff
@@ -14,13 +14,9 @@ def create_config(is_mc):
     config.General.transferLogs = True
 
     config.JobType.pluginName = 'Analysis'
-    config.JobType.psetName = '../python/dummy_pset.py'
-    config.JobType.scriptExe = '../bin/runFrameworkOnGrid.sh'
+    config.JobType.psetName = '../../Framework/test/TestConfiguration.py'
```

Comment:
I think this should be a command-line option to let the user specify which configuration he wants to use?